balpy/balpy.py

```diff
@@ -1543,6 +1548,50 @@ def balVaultGetInternalBalance(self, tokens, address=None):
 			internalBalances[token] = Decimal(balances[i]) * Decimal(10**(-decimals));
 		return(internalBalances);
 
+	def balVaultGetPool(self, poolId):
+		vault = self.balLoadContract("Vault")
+		address, specialization = vault.functions.getPool(poolId).call()
+		return address, specialization
+
+	def balVaultGetPoolTokenInfo(self, poolId, tokenAddress):
+		vault = self.balLoadContract("Vault")
+		tokenInfo = vault.functions.getPoolTokenInfo(poolId, tokenAddress).call()
+		cash = tokenInfo[0]
+		managed = tokenInfo[1]
+		lastChangeBlock = tokenInfo[2]
+		assetManager = tokenInfo[3]
```

Comment:
could also be 
```suggestion
		cash, managed, lastChangeBlock, assetManager, *_ = tokenInfo
```

Reply:
Yes! My idea was to make clear to the user how to parse the output. Also, is the way that is done in the other methods. I wanted to keep this structure for now